tox.ini

```diff
@@ -10,6 +10,37 @@ requires =
 tox_pip_extensions_ext_venv_update = true
 tox_pyenv_fallback = false
 
+[pycodestyle]
+ignore =
+    # Disable pycodestyle errors and warnings that we don't care about because
+    # Black formats our code for us.
+    E203,  # Whitespace before ':',
+    E231,  # Missing whitespace after ',',
+    E501,  # Line too long,
+    W503,  # Line break before binary operator,
+
+    # Bare except. PyLint finds these for us so we don't need pycodestyle to.
+    E722,
```

Comment:
pycodestyle's config can either go in `setup.cfg` or in `tox.ini`, it doesn't have its own separate config file, and we don't have a `setup.cfg`, so it's in here. This doesn't mean it has to be run in tox though -- pycodestyle run outside of tox style reads the `[pycodestyle]` in `tox.ini`

Reply:
Is there any harm in having a `setup.cfg`? Having two files might make each easier to read.

Reply:
I've moved these into `setup.cfg` in [the other pr](https://github.com/hypothesis/lms/pull/1523)